src/Saml/ResponseContext.php

```diff
@@ -27,113 +27,70 @@
 
 final class ResponseContext implements ResponseContextInterface
 {
-    private $hostedIdentityProvider;
-    private $stateHandler;
-    private $serviceProviderRepository;
-
     public function __construct(
-        IdentityProvider $identityProvider,
-        ServiceProviderRepository $serviceProviderRepository,
-        StateHandlerInterface $stateHandler
+        private IdentityProvider $hostedIdentityProvider,
```

Comment:
No `readonly` here?

Reply:
Think the rector rules were configured for up until PHP 8.1. I'm not going to add php82 features now.

Reply:
Keep this in mind for tomorrow :-)